src/openforms/forms/tasks.py

```diff
@@ -14,6 +15,23 @@
 logger = logging.getLogger(__name__)
 
 
+def on_variables_bulk_update_event(form_id: int) -> None:
+    """
+    Celery chain of tasks to execute on a bulk update of variables completion event.
+    """
+    recouple_reusable_definition_variables_task = (
+        recouple_reusable_definition_variables_to_form_variables.si(form_id)
+    )
```

Comment:
```suggestion
    repopulate_reusable_definition_variables_task = (
        recouple_reusable_definition_variables_to_form_variables.si(form_id)
    )
```
perhaps? it's not necessarily fixing broken relations, rather it's deriving the variables from the new FD again :thinking: